curriculum/schema/challengeSchema.js

```diff
@@ -81,6 +81,7 @@ const schema = Joi.object()
     question: Joi.object().keys({
       text: Joi.string().required(),
       answers: Joi.array().items(Joi.string()).required(),
+      assignments: Joi.array().required(),
```

Comment:
This is only true for the Odin project, so I think it would make more sense to separate the two rather that adding an unused property to video challenges.

So, `question` can stay as it is and `assignments` can be an optional property on the same level as `question`